presto-singlestore/src/test/java/com/facebook/presto/plugin/singlestore/TestSingleStoreTypeMapping.java

```diff
@@ -95,12 +95,12 @@ public void testBasicTypes()
     public void testPrestoCreatedParameterizedVarchar()
     {
         DataTypeTest.create()
-                .addRoundTrip(stringDataType("varchar(10)", createVarcharType(255 / 3)), "text_a")//utf-8
-                .addRoundTrip(stringDataType("varchar(255)", createVarcharType(255 / 3)), "text_b")
-                .addRoundTrip(stringDataType("varchar(256)", createVarcharType(65535 / 3)), "text_c")
-                .addRoundTrip(stringDataType("varchar(65535)", createVarcharType(65535 / 3)), "text_d")
+                .addRoundTrip(stringDataType("varchar(10)", createVarcharType(10)), "text_a")//utf-8
+                .addRoundTrip(stringDataType("varchar(255)", createVarcharType(255)), "text_b")
+                .addRoundTrip(stringDataType("varchar(21844)", createVarcharType(21844)), "text_c")
+                .addRoundTrip(stringDataType("varchar(21846)", createVarcharType(16777215 / 3)), "text_d")
                 .addRoundTrip(stringDataType("varchar(65536)", createVarcharType(16777215 / 3)), "text_e")
-                .addRoundTrip(stringDataType("varchar(16777215)", createVarcharType(16777215 / 3)), "text_f")
+                .addRoundTrip(stringDataType("varchar(16777215)", createVarcharType((int) (4294967295L / 3))), "text_f")
```

Comment:
Why are we using math expressions like `4294967295L/3` and why are we using `4294967295L` ?